### ISSUE TITLE
Transition to AndroidX

### ISSUE THREAD

**Issue by leofirespotter** [matthew-carroll/hover#85]
We are transitioning our app to AndroidX and I think because we have forked and are using it as a project instead of a library, hover is having issues.

Would you accept a PR that moves hover over to use AndroidX?

**Comment by md-lotfi** [comment]
Yes, we're waiting for such a transition, but the project hasn't been update for 2 years.
